<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1271</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tblMercaderia">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledBusqueda">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOrdenar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ordenar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerrar">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>490</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCargarEnCarrito">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>490</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cargar 
en carrito</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVender">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>490</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vender</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameAdmin">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>520</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnAgregar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>55</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Agregar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnModificar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>0</y>
       <width>55</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminarFila">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>55</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Eliminar</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnActualizar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>510</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiarCarrito">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>490</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Limpiar
 carrito</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblCarrito">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>40</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Columna:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>490</y>
      <width>70</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Eliminar
 Fila</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPrueba">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>110</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Disponible: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>450</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Importe total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTotal">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>450</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>$0.00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblDisponible">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>10</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="cboCondicion">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>500</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Condicion</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblColumna">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>500</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>471</width>
      <height>561</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>561</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 170, 127);</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>tblMercaderia</zorder>
   <zorder>ledBusqueda</zorder>
   <zorder>btnOrdenar</zorder>
   <zorder>btnCerrar</zorder>
   <zorder>btnCargarEnCarrito</zorder>
   <zorder>btnVender</zorder>
   <zorder>frameAdmin</zorder>
   <zorder>btnActualizar</zorder>
   <zorder>btnLimpiarCarrito</zorder>
   <zorder>tblCarrito</zorder>
   <zorder>label_3</zorder>
   <zorder>btnEliminar</zorder>
   <zorder>lblPrueba</zorder>
   <zorder>label_2</zorder>
   <zorder>lblTotal</zorder>
   <zorder>lblDisponible</zorder>
   <zorder>cboCondicion</zorder>
   <zorder>label_4</zorder>
   <zorder>lblColumna</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1271</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdministrar">
    <property name="title">
     <string>Administrar</string>
    </property>
    <addaction name="menuAdministrarProveedores"/>
    <addaction name="menuAdministrarUsuarios"/>
    <addaction name="menuAdministrarVentas"/>
   </widget>
   <widget class="QMenu" name="menuUsuario">
    <property name="title">
     <string>Usuario</string>
    </property>
    <addaction name="menuUsuarioCambiarPass"/>
   </widget>
   <addaction name="menuAdministrar"/>
   <addaction name="menuUsuario"/>
  </widget>
  <action name="actionAgregar_Vendedor">
   <property name="text">
    <string>Vendedores</string>
   </property>
  </action>
  <action name="menuAdministrarProveedores">
   <property name="text">
    <string>Proveedores</string>
   </property>
  </action>
  <action name="menuAdministrarUsuarios">
   <property name="text">
    <string>Usuarios</string>
   </property>
  </action>
  <action name="menuAdministrarClientes">
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="menuAdministrarVentas">
   <property name="text">
    <string>Ventas</string>
   </property>
  </action>
  <action name="menuUsuarioCambiarPass">
   <property name="text">
    <string>Cambiar contraseña</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
